src/js/shield_canvas_draw.js

```diff
@@ -35,6 +35,35 @@ export function paBelt(fillColor, strokeColor) {
   return ctx;
 }
 
+// Special case for Branson color-coded routes, documented in CONTRIBUTE.md
+export function bransonRoute(fillColor, strokeColor) {
+  var ctx = roundedRectangle(
+    Color.shields.green,
```

Comment:
I suppose the shield is analogous to the Allegheny County belts or the Chicago Skyway. The green part of the sign may not add much to the shield’s recognizability, but it keeps the shield anchored, so the color swatch doesn’t feel like it’s floating by itself. Have you tried out how the shield would look with balanced top and bottom padding?

Reply:
It's a judgment call, though I prefer the bottom-aligned version.

![Screenshot from 2022-08-05 12-25-32](https://user-images.githubusercontent.com/1732117/183120318-b2cb7e83-bf72-46f8-9153-e240fc950c03.png)